src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Colors {

```

Comment:
Let's use `Color` instead of `Colors`. See [here](https://stackoverflow.com/questions/15755955/naming-of-enums-in-java-singular-or-plural).